src/explorer.api/Explorers/CategoricalColumnExplorer.cs

```diff
@@ -44,7 +44,7 @@ public override async Task Explore(CancellationToken cancellationToken)
 
             var distinctValueCounts =
                 from row in distinctValuesQ.ResultRows
-                where !row.DistinctData.IsSuppressed
+                where !row.DistinctData.IsSuppressed && !row.IsNull
```

Comment:
Why are you using here `where !row.DistinctData.IsSuppressed && !row.IsNull`  and in some other places `where row.DistinctData.HasValue`?

Reply:
I did this rather quickly and some of the types don't implement `HasValue`. I need to go back through the changes and update the Type to implement HasValue but for now I just wanted to fix the error so that we could extract some useful data.

I've logged an issue to rectify this: #90 .